src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -1,48 +1,43 @@
-import {
-    MetadataInstanceForm,
-    type MetadataTemplateInstance,
-    UnsavedChangesModal,
-    type MetadataTemplate,
-} from '@box/metadata-editor';
+import { AutofillContextProvider, MetadataInstanceForm, type MetadataTemplateInstance } from '@box/metadata-editor';
+import noop from 'lodash/noop';
 import React from 'react';
 
 export interface MetadataInstanceEditorProps {
     isBoxAiSuggestionsEnabled: boolean;
     isUnsavedChangesModalOpen: boolean;
-    template: MetadataTemplateInstance | MetadataTemplate;
+    template: MetadataTemplateInstance;
+    onCancel: () => void;
 }
 
 const MetadataInstanceEditor: React.FC<MetadataInstanceEditorProps> = ({
     isBoxAiSuggestionsEnabled,
     isUnsavedChangesModalOpen,
     template,
+    onCancel,
 }) => {
     const handleSubmit = () => {
         // TODO in a future PR
     };
     const handleCancel = () => {
-        // TODO in a future PR
+        onCancel();
     };
     const handleDelete = () => {
         // TODO in a future PR
     };
 
     return (
-        <>
+        <AutofillContextProvider isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}>
```

Comment:
Provider is really needed only for editing templates. In later PR I'll add proper connection to API when we will have adapter ready.